electrum/commands.py

```diff
@@ -641,7 +644,10 @@ async def paytomany(self, outputs, fee=None, feerate=None, from_addr=None, from_
             rbf=rbf,
             password=password,
             locktime=locktime)
-        return tx.serialize()
+        result = tx.serialize()
+        if addtransaction:
+            await self.addtransaction(result)
```

Comment:
```suggestion
            await self.addtransaction(result, wallet=wallet)
```

Reply:
Hmm, I tested that on CLI with regtest and it worked, approximately like so:
`./run_electrum daemon start`
`./run_electrum load_wallet`
and then I just used
`./run_electrum payto x y`
and checked that it was added via `./run_electrum onchain_history`.
Didn't give errors. I think it should work because of daemon passing arguments (command decorator), or am I missing something?

Reply:
I had tested on master and it did not work.
The command decorator cannot set the wallet arg correctly in this case as it mostly does its magic based on kwargs which will be empty in this case.

Reply:
Ok, will apply the fixes.